src/adapters/RoomsJSONAdapter.test.js

```diff
@@ -94,6 +94,17 @@ describe('Rooms JSON Adapter Interface', () => {
     });
   });
 
+  test('getRoomActivities() converts date strings into date objects for time rulers', (done) => {
+    const timeRulerRoomID = 'time-rulers';
+
+    roomsJSONAdapter.getRoomActivities(timeRulerRoomID).subscribe((data) => {
```

Comment:
I would prefer three test cases

Reply:
The test verifies the date is being correctly converted. We don't need a test case for every `expect` statement.

Reply:
IMO the last expect statement covers the other two statements: 
```js
expect(data[0].date instanceof Date).toBeTruthy();
```
in order for `data[0].data` to be a `Date` instance, it must:
 1) be an object 
 2) be defined

Therefore, to me the first two expect statements are unnecessary to check.

Reply:
It helped when developing to know what part failed, but I will remove it.